frame/lending/src/mocks_offchain/wrapper.rs

```diff
@@ -0,0 +1,150 @@
+use serde::{Deserialize, Serialize};
```

Comment:
may be rename wrapper to more specific wrapper.

Reply:
Done